docs/src/guides/hub-search.md

```diff
@@ -0,0 +1,220 @@
+            "not": ["rivers"]
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+This reads as:
+
+- search for items
+  - of type `Feature Service`
+  - tag `authoritative`
+  - AND tag `water` or `lake`
+  - WITHOUT tag `rivers`
+
+We can combine predicates via the `.operation` to construct more complex queries
+
+```json
+{
+  "entityType": "item",
+  "filters": [
+    {
+      "operation": "OR",
+      "predicates": [
+        {
+          "type": "Feature Service",
+          "tag": {
+            "all": ["authoritative"],
+            "any": ["water", "lake"],
+            "not": ["rivers"]
+          }
+        },
+        {
+          "type": "Web Map",
+          "tag": {
+            "all": ["authoritative"],
+            "any": ["wildfire", "fire"]
+          },
+          "owner": "jsmith"
+        }
+      ]
+    }
+  ]
+}
+```
+
+This reads as:
+
+- search for items
+  - of type `Feature Service`
+  - tag `authoritative`
+  - AND tag `water` or `lake`
+  - WITHOUT tag `rivers`
+  - OR
+  - of type `Web Map`
+  - tag `authoritative`
+  - AND tag `wildfire` or `fire`
+  - owned by `jsmith`
+
+## Predicate Properties
+
+Although the `IPredicate` structure allows for any key, value combinations, the following table lists the properties and their allowed types, for queries being serialized to the ArcGIS Portal API.
+
+| Property           | Entity                  | Allowed Types                         |
+| ------------------ | ----------------------- | ------------------------------------- |
+| `access`           | `item`, `group`, `user` | `string`, `string[]`, `IMatchOptions` |
+| `categories`       | `item`                  | `string`, `string[]`, `IMatchOptions` |
+| `created`          | `item`, `group`, `user` | `IDateRange<number>` `IRelativeDate`  |
+| `description`      | `item`, `group`         | `string`, `string[]`, `IMatchOptions` |
+| `disabled`         | `user`                  | boolean                               |
+| `email`            | `user`                  | `string`, `string[]`, `IMatchOptions` |
+| `emailstatus`      | `user`                  | `string`, `string[]`, `IMatchOptions` |
+| `firstname`        | `user`                  | `string`, `string[]`, `IMatchOptions` |
+| `fullname`         | `user`                  | `string`, `string[]`, `IMatchOptions` |
+| `group`            | `item`, `user`          | `string`, `string[]`, `IMatchOptions` |
+| `id`               | `item`, `group`         | `string`, `string[]`, `IMatchOptions` |
+| `isInvitationOnly` | `group`                 | boolean                               |
+| `isopendata`       | `group`                 | boolean                               |
+| `joined`           | `group`                 | `IDateRange<number>` `IRelativeDate`  |
+| `lastlogin`        | `user`                  | `IDateRange<number>` `IRelativeDate`  |
+| `lastname`         | `lastname`              | `string`, `string[]`, `IMatchOptions` |
```

Comment:
I assume that the second `lastname` should be `user`, right?

Reply:
doh! Thanks